### PR TITLE
fix: adds v3 ethereum protocol addresses. adds patch to take into acc…

### PR DESCRIPTION
**Pull request by sendra**
…ount pools that have no reserves

### CODE REVIEW

File: scripts/config.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export enum ChainId {
 }
 
 const RPC_PROVIDERS = {
-  [ChainId.mainnet]: "https://rpc.flashbots.net",
+  [ChainId.mainnet]: 'https://rpc.tenderly.co/fork/1e8e167f-4a45-4ac5-8ce1-4045b07a0faf',//"https://rpc.flashbots.net",
```
Comment (eboadom) [inline scripts/config.ts:23]: Which is the rationale of this?

File: scripts/config.ts [hunk 2]
```diff
@@ -289,6 +289,15 @@ export const pools: Pool[] = [
       FAUCET: "0xC52eA1F19C22E5a3725105BC0cf4988614e84D98",
     },
   },
+  {
+    name: "AaveV3Ethereum",
+    chainId: ChainId.mainnet,
+    addressProvider: "0x2f39d218133AFaB8F2B819B1066c7E434Ad94E9e",
+    version: 3,
+    // additionalAddresses: {
```
Comment (eboadom) [inline scripts/config.ts:297]: This should be removed

File: scripts/config.ts (hunk 1 above)
Comment (sendra) [reply scripts/config.ts:23]: to get the addresses on the v3.0.1 deployed on tenderly.

File: scripts/config.ts (hunk 2 above)
Comment (sendra) [reply scripts/config.ts]: no need added here all the extra addresses that are on deployment but we cant bia rpc right now

**Review by eboadom — APPROVED** [review]

**Pull request merged**